### DISCUSSION TITLE
求激活，电脑更新后要重复激活，谢谢。

### DISCUSSION THREAD

**Discussion by deepblue1212** [LFTEC/LearningFucker#39] [General]
请将下面代码复制到Github讨论区中，收到激活码后，调用activate命令进行激活
------------------------------License------------------------------
pfCWOMDVy3Mx9NBybzclby5yQq9AHELhnvjxQpzfVBB8EzdMkeYhi7odvCbrpu5zDwr
sfRmgN84TNtDWGYk67LsBkqD+3IYWxBS/KoE6wcRymR3esSA0yAjWFXoUdTk3onbvFj
2Y2TBUlAm7oFLOKNPPg+ayAnnKjEl2pmRcmGKIfpLQrOCegIOZIpOp+Lqg/cQv4gwUK
2A7dcT3pNUXpSmrX/u7QtLrOEUXuM8NHCUzEOA8AYJ5RHQdy4qmiM/0hII2kqxes6mg
82uFAFU6QHEP5Jox4sWG9YQdHnV4E3qpXKuqKAVSX/41DNxfjc3JaKSrK+XgEgwwgC5
rT7Tsz/jtoxC/IDE23FjOlcpCHHBX/zQ++ZtEJEydEtywCqZi19sv3c+vzjjLoAM4ev
7LVPPKZk6qzz+b6ZcIGpmLc/rYDPkw9Mv1LJW6VO0ftQNAR94749KQeOXDjLBxZyTew
WvVDL6OsnWn0DgjiKiZLTF3teB8d2Aw2NXMPZiYtvs9KQ6fv6WtkO5li0+XxdcAJ6cL
B0O0nT9Eq7iP3YM2Is2lXXT0wcebdQJABOvSJuYLKglXsIW7lvdNHXhJsLjusr192xQ
kxsYml/lWT5NL0Z432a8HATej9RsYcnel39dvp7knP+U8ALgUQAmZHYQa4M1NdiYFxl
tI0OlXhkjjWoseULWWA9NqBl9xIhzh1R273gc2TOgoDelU1OIoKTlIrTwLedhpsmnEJ
6wIeS6t1ENnbNDB1tDQ7pwKhegUlRM0HLS9h+dLcJxrTnyy9MsRdbgMewaIf+bH2WPD
QuBYMu4v++czZTQ035C+AgHzkfRPdeF1qx0rObvGkb8+6biIZDwAKVMzVviA5IgJssg
IRA+9VqtU7KtjGrNVf6zDtYPUH7Z7amWzAOUNdadfklkvzF1aXzs3kNMjP66q0vX5+6
UOCt/5aReOtKnwR7AlpRNds3SlmKzCrM4+2cFZGE/A6Ns0wuSvESnglOQg0vE75vGUz
qVWqBET0e5X9gNbzF/X1BhvXXQGzC4Oyq+V/KNnBFGssA13ys5bl4WNwGZ2bfzZ5gT5
bvo6FP1GAs/raIL9ViYhuJ1Pr8F0GSh5TQoMlf+vhb5j6bB7e0ToimbIa0j3QvVxkSX
S7V5dn4QF+32MVy4KaXPUx3d90D2N3JlYmpMRsU8C7grOik+ppa8FmuxX8Z+YHxJgb9
hbJEHZ4BM2EuJoG0XtFzE1Zu1xZtWt4sU8lg8nI9dcfsMr8/v1Qoi4j1LKH5L46HGiq
LA3trbx8dqM6NdRanN4mSeufjMmJsRCrTzf26WUA60bdmhWHPwabHUXBf8/iUbOWDSU
IKdj+apguzZ+agN8wwqeuyBimA==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
TY1edivRVbtgyo48Cvwhe6OreW/LAFEcXmP45AO6aHyGBYBsIh1zM5CmY7jCM+isI6urAmOlmvjN00ItTbtm4NZ2bC1KS7rGZ29RxNSV+vLTSABiBl6wkzJEFOr5q/Ro65/cLzgD59MxXf+o4Q/cBbK0BLespVG9AGFgRHpZAL3rqDBzUI6n0BwYCLJdt0I3s1YQa+c1sIXTOrilq7T1Da2FOPtJnRW83c1Se3tz4KKNJ2X8Myq+AOOC7NSouxiauxSfp/sl8hCg00wHidbqakV9efbJtz42QM4KfdkrGDDC7qmBGyz3IVX292LO3A2uge97mXwsvx3TYCoV1t7K7DU41uOXnxuq3H2RnVgPug7WywhQ5RdMt1HgXZWP0JrohHmlmsyenOaL7v0qGDm23MADLO5yn7FY1ves1lyExuz5kcHLvynnDN5y6FoGBvxr2OcKet5Q73SqNbqRwQmwp77kG+F31tQS5UA8MS5wRRv0o4QHMl8xJuZukfKBYEI+fgvi4cSx2vhpEwWAdJlsuAvWOVtB4v6+phfiqEXu7ZGs08EhBc1aWejBEj9gCzCC3txQ/SypsaKcTBXJR9hZxnCPl5y1NJPBTfY4V1iV1ahjsz7bNFL79k/N2DmcAepz5ubIGlUPtgjxj6HfZzC0AgNZG1Tr/exRy7QJlBAkAhlCMj9BO2HJkJ3RM/AV3dHgYQemJvQUqlruaM9l3vXoRjjl5gIEXteZbHEvInuKdloH0RO1tQkON5cR3TPj3Xjax43Ug9pUktVabOyJi7fSDbsq6BRV9ZoBF0iq726bl7JVHOIH/WNaOlXJdvgRRaLJ17uzxsMD7M5LlnsGgWCJPIwvTQ0rxbzA8OmF+aYw/zZtlbV2/FLqYhaD1AFsfAB/wZ1jqYx1wCZc1kjsjAQm76MGmCvqAeN4DpcB412zcSuACMSOugas3G4q9f3MqvCo/QNORzLm6NT2vpGgOXq9DRCXV1wiAxGMpUM78O27iULDTD7KoGnnjnzJGPyU3XEAferb3U7nFD3uzOqSt4+2lzB1Kfe3xrYQJBjV6MuimPMqkSfVN8mC9lp9keKNBTja5DoYWzKJf2pvoI4pXtC+QdGmR1lxKJS8ar+2cHeNLRcW7Q9jgmAGJW3JkB9h9SAn7E02+0p3blxhCpdSbVUXja3H7bKhn0zNAy0WzAGngJ60svyS7IDEsY8vgPCsvIkHXkYbm9A4BX+1oah2X3e+q7x2fF00O+uBoal8c1Cx5JfBaSBkZGKk0ugEEr860r/ym7s7yr2nzcEAHdKw4ecFBxsEpuu3WHjVXCF5NglJaiyRL/cYuYvYIP8Ouh9AgJi1cFIebnTYT+NWmW1fxl+m3w==
